jmh-benchmarks/src/main/java/org/apache/kafka/jmh/acl/AuthorizerBenchmark.java

```diff
@@ -204,17 +211,17 @@ public void tearDown() throws IOException {
     }
 
     @Benchmark
-    public void testAclsIterator() {
-        authorizer.acls(AclBindingFilter.ANY);
+    public Iterable<AclBinding> testAclsIterator() {
+        return authorizer.acls(filter);
     }
 
     @Benchmark
-    public void testAuthorizer() {
-        authorizer.authorize(authorizeContext, actions);
+    public List<AuthorizationResult> testAuthorizer() {
+        return authorizer.authorize(authorizeContext, actions);
     }
 
     @Benchmark
-    public void testAuthorizeByResourceType() {
-        authorizer.authorizeByResourceType(authorizeByResourceTypeContext, AclOperation.READ, ResourceType.TOPIC);
+    public AuthorizationResult testAuthorizeByResourceType() {
+        return authorizer.authorizeByResourceType(authorizeByResourceTypeContext, op, resourceType);
```

Comment:
alternatively, we could use a blackhole. Won't this show up as a warning "unused parameter" in INtelliJ?